pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -207,6 +207,17 @@ class Config11 {
                         nightly: ['sanity.openjdk'],
                         weekly : ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf']
                 ],
+        ],
```

Comment:
Should we be adding aarch64Windows to jdk11u in this PR?

Reply:
yeah, it is in the line 212 (below)